docs/components/console/manage-organization/alpha-terms.md

```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
+title: Alpha terms
```

Comment:
Could we adjust the title to something more product-oriented and less legal-sounding? "Enable alpha features" or similar? 

"Alpha terms" sounds like something I would find in the footer of a website next to the policies.

@HanselIdes is there any relation to "alpha releases" (our monthly release assets)?

Reply:
@akeller this page refers to the program to enable alpha features in Console and Modeler. Alpha Terms is probably too legal-ese for it

Reply:
Adjusted in latest commit 👍